src/main/java/hudson/plugins/git/util/GitUtils.java

```diff
@@ -53,6 +53,35 @@ public static Node workspaceToNode(FilePath workspace) { // TODO https://trello.
     }
 
     /**
+     * Call close method on walk using reflection.  JGit 3 uses
```

Comment:
:ant: from the method javadoc I would expect that this method would call JGit 3's release for me... which would seem like a nice thing for the method to do... instead we have the method call direct at the call site...